configuration/index.md

```diff
@@ -133,5 +133,6 @@ Every interface has it's own strong points.
 With openHAB 1.x the smart home configuration was done via configuration files only.
 One of the most visible additions to openHAB 2 is the adminstrative web interface Paper UI.
 
-Things and Items can either be defined and managed in configuration files or handled by Paper UI in a system-side database.
-Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
+Things and Items can either be defined and managed in configuration files or handled by Paper UI in a system-side database. The database used by Paper UI is a human readable (and editable for advanced users) [JSON database]({{base}}/administration/jsondb.html) found in the `userdata` folder of your openHAB installation.
```

Comment:
Please make a newline here.
We keep the docs to one sentence per line, to make maintaining them easier.